hipcub/PKGBUILD

```diff
@@ -2,46 +2,29 @@
 pkgname=hipcub
 pkgver=3.3.0
 pkgrel=2
-pkgdesc="Reusable software components for the rocm developers."
+pkgdesc='header-only library on top of rocPRIM or CUB'
 arch=('x86_64')
 url="https://github.com/ROCmSoftwarePlatform/hipCUB"
```

Comment:
Should the url be changed from the development page to the documentation page: https://rocm-documentation.readthedocs.io/en/latest/ROCm_Libraries/ROCm_Libraries.html#hipcub?

Reply:
This change would apply to almost all packages of the ROCm stack. Right now, $url always points to the git repo.

Reply:
I think the plan is to point to the official documentation. See: https://github.com/rocm-arch/rocm-arch/pull/72#issuecomment-604835787 and the following comment.